schemas/sitePreferencesSchema.json

```diff
@@ -151,14 +151,14 @@
         "baseLayer": {
           "type": "string",
           "title": "Base Layer",
-          "default": "blueMarble",
+          "default": "worldImagery",
```

Comment:
Ensure that `blueMarble` is temporarily valid so the user will never see an error message on the preferences page